pyscriptjs/src/utils.ts

```diff
@@ -14,8 +14,12 @@ function getLastPath(str: string): string {
     return str.split('\\').pop().split('/').pop();
 }
 
+function escape(str: string): string {
+    return str.replace(/</g, "&lt;").replace(/>/g, "&gt;")
+}
+
```

Comment:
I think we should escape more?
I'm not an expert in the field, but a quick googling found this:
https://stackoverflow.com/a/6234804

I guess we should probably escape `'`, `"` and `&` as well?

Reply:
I was indeed quite conservative here, however I think `<` and `>` may indeed be special in the regard that they absolutely break the parser while the others are generally parsed correctly. Might be best to simply write some tests to confirm.

Reply:
for example, imagine the following code:
```html
<py-script>
js.console.info("a &amp; b");
</py-script>
```

I would expect it to print literally `a &amp; b`, what it actually prints is `a & b`.
And if you try to print `"a &quot b"` is even worse, because it is parsed as a quote `"` and so python read `"a " b"`, which results in a python `SyntaxError`.

Reply:
That's indeed bad, sounds like we actually have to **un**escape those HTML entities.

Reply:
uhm right, for those it's the opposite direction.
Btw, I just checked what JS does:
```html
<script>
    console.info("a &amp; b");
</script>
```
prints `a &amp; b`, so we should probably do the same.